cinema/views.py

```diff
@@ -0,0 +1,45 @@
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from cinema.serializers import MovieSerializer
+from django.shortcuts import get_object_or_404
+
+from cinema.models import Movie
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    movies = Movie.objects.all()
+    if request.method == "GET":
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    if request.method == "POST":
+        serializer = MovieSerializer(data=request.data)
+
+        if serializer.is_valid():
+            serializer.save()
```

Comment:
You can pass raise_exception=True argument to is_valid.
https://www.django-rest-framework.org/api-guide/exceptions/#validationerror